csharp_dotnetcore/12.NLP-With-LUIS/LuisBot.bot

```diff
@@ -0,0 +1,38 @@
+﻿{
+    "name": "LuisBot",
+    "description": "LUIS Bot Sample",
+    "secretKey": "",
+    "services": [
+      {
+        "id": "http://localhost:3978/api/messages",
+        "name": "LuisBot",
+        "type": "endpoint",
+        "appId": "",
+        "appPassword": "",
+        "endpoint": "http://localhost:3978/api/messages"
+      },
+      {
```

Comment:
This would not work with the MSBOT CLI. Please ensure that having blank sections like this in the .bot file would actually work.

Reply:
Ah, I think I see what you're saying - you want to begin with a blank .bot file and then use the cli to manipulate and add the sections.

In the spirit of simplicity, is downloading another optional CLI tool the best approach?  If so, should modify the pre-requisites to make the tools (and npm/etc) mandatory as pre-requisites for bot development.